MediaRecorder.html

```diff
@@ -340,149 +340,87 @@
+    <h2>Error Handling</h2>
 
+    <section id="general-principles">
+      <h3>General Principles</h3>
 
+      <p>Errors are indicated via <a href="http://www.w3.org/TR/2012/WD-dom-20121206/#exception-domexception">DOMException</a>s raised (see [[!DOM4]]). An exception <em title="must" class="rfc2119">must</em> be thrown when the error can be detected at the time that the call is made. If recording has been started and not yet stopped when the error occurs, then after raising the error, the UA <em title="must" class="rfc2119">must</em> raise a  dataavailable event, containing any data that it has gathered, and then a stop event. The UA  <em title="may" class="rfc2119">may</em> set platform-specific limits, such those for the minimum and maximum Blob size that it will support, or the number of Tracks it will record at once.  It <em title="must" class="rfc2119">must</em> signal a fatal error if these limits are exceeded. </p>
+    </section>
 
+    <section id="exception-summary" class="informative">
+      <h3>Exception summary</h3>
+      <p> Each of the exceptions defined in this document is a <a href="http://www.w3.org/TR/2012/WD-dom-20121206/#exception-domexception"><code>DOMException</code></a> with a specific type. The exception types and properties such as code value are defined in [[!WEBIDL]]. </p>
 
-      <section id="MediaRecorderErrorEvent">
-          <h2>MediaRecorderErrorEvent</h2>
-          <dl class="idl" title="interface MediaRecorderErrorEvent : Event">
-              <dt>readonly attribute RecordingErrorName name</dt>
-              <dd>The name of the error</dd>
-              <dt>readonly attribute DOMString? message</dt>
-              <dd>A UA-dependent string offering extra human-readable information about the error.</dd>
-         </dl>
-</section>
+      <table class="vert">
+        <thead>
+          <tr><th>Name</th><th>Description</th></tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><code>InvalidStateError</code></td>
+            <td> An operation was called on an object on which it is not allowed or at a time when it is not allowed, or if a request is made on a source object that has been deleted or removed. </td>
+          </tr>
+          <tr>
+            <td><code>UnknownError</code></td>
+            <td>The operation failed for an unknown transient reason (e.g. out of memory). User agents SHOULD provide as much additional information as possible in the <code>message</code> attribute.</td>
+          </tr>
+          <tr>
+            <td><code>NotSupportedError</code></td>
+            <td>A modification to the <code>stream</code> has occurred that makes it impossible to continue recording. An example would be the addition of a Track while recording is occurring. User agents SHOULD provide as much additional information  as possible in the <code>message</code> attribute.</td>
+          </tr>
+          <tr>
+            <td><code>SecurityError</code></td>
```

Comment:
Searching in the document (your branch opened in a browser), SecurityError is the only exception I see referenced outside of this section. This section is non-normative, so some other part of the spec needs to say when these exceptions are thrown.